### DISCUSSION TITLE
Adjustment of the "ownership rights" in character sheet

### DISCUSSION THREAD

**Discussion by SwedishRabbit** [fvtt-fria-ligan/forbidden-lands-foundry-vtt#329] [General]
Is it possible to have one of the levels between "owner" and "None" able to do only dice rolls and edit in Bio and Notes

It would enable player engagement and cooperation

**Comment by aMediocreDad** [comment]
I am not really interested in doing this. If you need someone to make rolls for another person then make them owner.

There are a whole lot of things going on behind the scenes and rolls may result in the character having reduced stats etc. It becomes very hairy when you only want to allow such edits in certain scenarios. 

I don't really see what the harm is in making everyone owners in those instances where you want collab.

**Reply by SwedishRabbit** [reply]
Okay, I understand. Yes i am going to make them owners of the character sheets. Thanks for the reply and the good work! Cheers!
